src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSecurity.cs

```diff
@@ -291,7 +291,7 @@ public sealed class FileSystemAuditRule : AuditRule
         private static int AccessMaskFromRights(FileSystemRights fileSystemRights)
         {
             if (fileSystemRights < (FileSystemRights)0 || fileSystemRights > FileSystemRights.FullControl)
-                throw new ArgumentOutOfRangeException(nameof(fileSystemRights), SR.Format(SR.Argument_InvalidEnumValue, fileSystemRights, "FileSystemRights"));
+                throw new ArgumentOutOfRangeException(nameof(fileSystemRights), SR.Format(SR.Argument_InvalidEnumValue, fileSystemRights, nameof(fileSystemRights)));
```

Comment:
This is not correct.  The last argument is supposed to be the type name of the enum, not the parameter name.